sdk/keyvault/Azure.Security.KeyVault.Keys/src/JsonWebKey.cs

```diff
@@ -120,6 +120,250 @@ public JsonWebKey(string keyId, string keyType, IList<string> keyOperations)
         /// HSM Token, used with "Bring Your Own Key"
         /// </summary>
         public byte[] T { get; set; }
+
+        internal override void ReadProperties(JsonElement json)
+        {
+            if (json.TryGetProperty("kid", out JsonElement kid))
+            {
+                KeyId = kid.GetString();
+            }
+            if (json.TryGetProperty("kty", out JsonElement kty))
+            {
+                KeyType = KeyTypeExtensions.ParseFromString(kty.GetString());
+            }
+            if (json.TryGetProperty("key_ops", out JsonElement key_ops))
+            {
+                foreach (var element in key_ops.EnumerateObject())
+                {
+                    KeyOps.Add(KeyOperationsExtensions.ParseFromString(element.Value.ToString()));
+                }
+            }
+            if (json.TryGetProperty("curveName", out JsonElement curveName))
+            {
+                CurveName = curveName.GetString();
+            }
+            if (json.TryGetProperty("n", out JsonElement n))
+            {
+                N = Encoding.ASCII.GetBytes(n.GetString());
+            }
+            if (json.TryGetProperty("e", out JsonElement e))
+            {
+                E = Encoding.ASCII.GetBytes(e.GetString());
```

Comment:
Same here. The Bas64 behavior should match what the comments say about these properties. Here/elsewhere.

If the data is expected to be Base64 encoded string, then it needs to be base64 decoded to a byte[], not ASCII-transcoded.

At least that's what I can gather from a quick review. Let me know if I am mistaken and the semantics are different here.